src/TrezorWallet.js

```diff
@@ -1,98 +1,175 @@
 import EthereumTx from 'ethereumjs-tx';
-import TrezorConnect from './trezorConnect';
+import { publicToAddress } from 'ethereumjs-util';
+import BigNumber from 'bignumber.js';
+import { timeout, TimeoutError } from 'promise-timeout';
+import HDKey from 'hdkey';
+const trezor = require('trezor.js');
+
+let currentSession = null;
+let currentDevice = null;
+const CUSTOM_TIME_OUT = 30000;
+
+const hardeningConstant = 0x80000000;
+const defaultAddress = [
+	(44 | hardeningConstant) >>> 0,
+	(60 | hardeningConstant) >>> 0,
+	(0 | hardeningConstant) >>> 0,
+	0
+];
+const deviceList = new trezor.DeviceList();
 
 export default class TrezorWallet {
-  constructor(networkId, path) {
+  constructor(networkId, accountsOffset = 0, accountsQuantity = 6, eventEmitter) {
     this.networkId = networkId; // Function which should return networkId
     this.getAccounts = this.getAccounts.bind(this);
     this.signTransaction = this.signTransaction.bind(this);
-    this.setDerivationPath = this.setDerivationPath.bind(this);
-    this.setDerivationPath(path);
+    this.accountsOffset = accountsOffset;
```

Comment:
Wallet class should not have offset/quantity parameters. It should be able to return all accounts, and the offeset/quantity parameter controlled by `getAccounts`

Reply:
@sk91 we cannot retrieve all accounts on creation of the class  because it is infinity based on the hardware wallet generating them. getAccount is a overwrite of the web3.eth.getAccounts method so web3 will call it only with the callback parameter. Cheers